### PR TITLE
Address self-review comments for Azure.ResourceManager.DesktopVirtualization

### PR DESCRIPTION
**Pull request by ArcturusZhang**
# Contributing to the Azure SDK

Please see our [CONTRIBUTING.md](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md) if you are not familiar with contributing to this repository or have questions.

For specific information about pull request etiquette and best practices, see [this section](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#pull-request-etiquette-and-best-practices).

### CODE REVIEW

File: sdk/desktopvirtualization/Azure.ResourceManager.DesktopVirtualization/src/autorest.md [hunk 1]
```diff
@@ -46,28 +46,53 @@ rename-rules:
...
+  HostPool.properties.ssoadfsAuthority: SsoAdfsAuthority
+  HostPool.properties.validationEnvironment: IsValidationEnvironment
+  HostPoolPatch.properties.ssoadfsAuthority: SsoAdfsAuthority
+  MsixPackage.properties.lastUpdated: LastUpdatedOn
+  ResourceModelWithAllowedPropertySet.managedBy: -|arm-id
+  SessionHost.properties.lastUpdateTime: LastUpdatedOn
+  SessionHost.properties.lastHeartBeat: LastHeartBeatOn
+  SessionHost.properties.resourceId: -|arm-id
+  SessionHost.properties.virtualMachineId: VmId
+  Workspace.properties.cloudPcResource: IsCloudPcResource
```
Comment (Yao725) [inline sdk/desktopvirtualization/Azure.ResourceManager.DesktopVirtualization/src/autorest.md:70]: Same as above

File: sdk/desktopvirtualization/Azure.ResourceManager.DesktopVirtualization/src/autorest.md [hunk 2]
```diff
@@ -46,28 +46,53 @@ rename-rules:
...
+  Status: SessionHostStatus
+  Operation: MigrationOperation
+  ExpandMsixImage.properties.lastUpdated: LastUpdatedOn
+  HealthCheckName: SessionHostHealthCheckName
+  HealthCheckResult: SessionHostHealthCheckResult
+  SSOSecretType: HostPoolSSOSecretType
```
Comment (Yao725) [inline sdk/desktopvirtualization/Azure.ResourceManager.DesktopVirtualization/src/autorest.md:76]: Should it be `Sso`?

File: sdk/desktopvirtualization/Azure.ResourceManager.DesktopVirtualization/api/Azure.ResourceManager.DesktopVirtualization.netstandard2.0.cs [hunk 3]
```diff
@@ -898,25 +839,66 @@ public partial class ScalingSchedule
...
+        public static Azure.ResourceManager.DesktopVirtualization.Models.SessionHostHealthCheckName DomainJoinedCheck { get { throw null; } }
+        public static Azure.ResourceManager.DesktopVirtualization.Models.SessionHostHealthCheckName DomainReachable { get { throw null; } }
+        public static Azure.ResourceManager.DesktopVirtualization.Models.SessionHostHealthCheckName DomainTrustCheck { get { throw null; } }
+        public static Azure.ResourceManager.DesktopVirtualization.Models.SessionHostHealthCheckName FSLogixHealthCheck { get { throw null; } }
+        public static Azure.ResourceManager.DesktopVirtualization.Models.SessionHostHealthCheckName MetaDataServiceCheck { get { throw null; } }
+        public static Azure.ResourceManager.DesktopVirtualization.Models.SessionHostHealthCheckName MonitoringAgentCheck { get { throw null; } }
+        public static Azure.ResourceManager.DesktopVirtualization.Models.SessionHostHealthCheckName SupportedEncryptionCheck { get { throw null; } }
+        public static Azure.ResourceManager.DesktopVirtualization.Models.SessionHostHealthCheckName SxSStackListenerCheck { get { throw null; } }
```
Comment (fengzhou-msft) [inline sdk/desktopvirtualization/Azure.ResourceManager.DesktopVirtualization/api/Azure.ResourceManager.DesktopVirtualization.netstandard2.0.cs:863]: SxS is one acromyn (Side by Side), the case should probably be `SxsStackListenerCheck `

File: sdk/desktopvirtualization/Azure.ResourceManager.DesktopVirtualization/api/Azure.ResourceManager.DesktopVirtualization.netstandard2.0.cs [hunk 4]
```diff
@@ -898,25 +839,66 @@ public partial class ScalingSchedule
...
+        public static Azure.ResourceManager.DesktopVirtualization.Models.SessionHostHealthCheckName UrlsAccessibleCheck { get { throw null; } }
+        public static Azure.ResourceManager.DesktopVirtualization.Models.SessionHostHealthCheckName WebRTCRedirectorCheck { get { throw null; } }
```
Comment (fengzhou-msft) [inline sdk/desktopvirtualization/Azure.ResourceManager.DesktopVirtualization/api/Azure.ResourceManager.DesktopVirtualization.netstandard2.0.cs:865]: WebRtcRedirectorCheck

**Review by fengzhou-msft — APPROVED** [review]

**Review by Yao725 — APPROVED** [review]

**Pull request merged**